python/bridgestan/__init__.py

```diff
@@ -1,4 +1,5 @@
 from .compile import compile_model, set_bridgestan_path
 from .model import StanModel
+from .__version import __version__
```

Comment:
Are you asking if we should be doing more, or less, in the init file? Having `bridgestan.__version__` defined is pretty common (e.g. https://github.com/numpy/numpy/blob/28bce82c82ba1151baeb7399abf454418b33eefe/numpy/__init__.py#L440) but I don't personally like adding it to `__all__` because I don't think `from bridgestan import *` should import `__version__` into global scope

Reply:
I was asking if we should add `__version__` to `__all__`, with the thought (from memory) that numpy does this.  But maybe my memory is incorrect here.

Your opinion makes sense to me.

And a user could still `from bridgestan import __version__` correct?

Reply:
Correct, a user can still manually import it or access it qualified (`import bridgestan; bridgestan.__version__`)

It seems the "big" packages are inconsistent with whether or not they put it in `__all__`. numpy and scipy do, pandas and matplotlib do not. I can't find anything "official" which states a preference.